integration-tests/itest/kctl/kctl.go

```diff
@@ -0,0 +1,84 @@
+package kctl
```

Comment:
[minor] I do not think it is very useful to shorten `kubectl` to `kctl`. It introduces a cognitive load, forcing people to get used to a new alias for a well-known tool. I suggest you rename this to `kubectl`.